### PR TITLE
Initial OCP CoreOS Layering/hotfixes docs

### PR DESCRIPTION
**Pull request by cgwalters**
This pairs with https://github.com/openshift/enhancements/blob/master/enhancements/ocp-coreos-layering/ocp-coreos-layering.md

(If someone who knows this build tooling and asciibinder could pick this up, that'd be great.  I kept finding myself writing in Markdown as I'm more familiar with it.  Happy to collaborate)

### CODE REVIEW

File: modules/installation-special-config-custom-image.adoc [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+== Applying the custom container image
+
+
+. Create a MachineConfig file which overrides `osImageURL`, referencing
+your custom image.
+
++
+----
+$ cat << EOF > ./99-masters-chrony-configuration.yaml
```
Comment (romfreiman) [inline modules/installation-special-config-custom-image.adoc:38]: chrony is missleading here if the example talks about replacing NM

File: modules/installation-special-config-custom-image.adoc [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+version, use `oc adm release info --image-for rhel-coreos-8 <image>`
+on the release image reported by `oc get clusterversion`.
+
+== Generating a container image with overrides
+
+This example `Dockerfile` demonstrates using `rpm-ostree override replace`
+to generate a container image that replaces the `NetworkManager` package:
+
+```
+FROM <operating system base>
```
Comment (romfreiman) [inline modules/installation-special-config-custom-image.adoc:22]: what should be written as operating-system-base? maybe add a comment with an example?

Comment (cgwalters) [reply modules/installation-special-config-custom-image.adoc:22]: This is the section above on finding the image, which is annoying and painful until we fix https://issues.redhat.com/browse/ART-4352

File: modules/installation-special-config-custom-image.adoc (hunk 1 above)
Comment (jmencak) [reply modules/installation-special-config-custom-image.adoc:38]: not only chrony, but `masters` too.

File: modules/installation-special-config-custom-image.adoc [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+RUN rpm-ostree override replace https://example.com/rhel-8-mirror//BaseOS/x86_64/os/Packages/NetworkManager-1.40.0-1.el8.x86_64.rpm \
```
Comment (jensfr) [inline modules/installation-special-config-custom-image.adoc:23]: && missing here at the end of the line?

**Pull request closed without merging**